08_4_softmax_pg_pong.py

```diff
@@ -53,7 +53,7 @@
 
 # Loss function
 # Sum (Ai*logp(yi|xi))
-log_lik = -Y * tf.log(action_pred)
+log_lik = -Y * (tf.log(action_pred)+0.0000000001)
```

Comment:
nan + any number is still nan though.

Shouldn't you add inside the log?
`tf.log(action_pred + 1e-8)`